lib/components/narrative/narrative-itineraries-header.js

```diff
@@ -97,12 +100,12 @@ export default function NarrativeItinerariesHeader ({
               onChange={onSortChange}
               value={sort.value}
             >
-              <option value='BEST'>Best option</option>
-              <option value='DURATION'>Duration</option>
-              <option value='ARRIVALTIME'>Arrival time</option>
-              <option value='DEPARTURETIME'>Departure time</option>
-              <option value='WALKTIME'>Walk time</option>
-              <option value='COST'>Cost</option>
+              <option value='BEST'>{intl.formatMessage({id: 'components.NarrativeItinerariesHeader.selectBest'})}</option>
+              <option value='DURATION'>{intl.formatMessage({id: 'components.NarrativeItinerariesHeader.selectDuration'})}</option>
+              <option value='ARRIVALTIME'>{intl.formatMessage({id: 'components.NarrativeItinerariesHeader.selectArrivalTime'})}</option>
+              <option value='DEPARTURETIME'>{intl.formatMessage({id: 'components.NarrativeItinerariesHeader.selectDepartureTime'})}</option>
+              <option value='WALKTIME'>{intl.formatMessage({id: 'components.NarrativeItinerariesHeader.selectWalkTime'})}</option>
+              <option value='COST'>{intl.formatMessage({id: 'components.NarrativeItinerariesHeader.selectCost'})}</option>
```

Comment:
Can `FormattedMessage` be used here instead?

Reply:
Same as previous